src/analytics/analytics-tracker-window.ts

```diff
@@ -1,9 +1,60 @@
+import { isFunction } from 'lodash';
+
 export interface AnalyticsTracker {
     track(step: string, data: any): void;
-    hit(hitName: string, data: any): void;
```

Comment:
What about the non-segment.com trackers that depend on `hit` and `hasPayloadLimit`?

Reply:
This PR removes dependency on this methods. After merging this PR we also remove `hit` and `hasPayloadLimit` methods from analytics.js added in DATA-6891.

Reply:
You can't remove `hit` and `hasPayloadLimit` from `analytics.js` due to the fact that there are clients using an older version of the SDK that references those methods.

Reply:
Fair enough, so... when we can delete this methods from analytics.js? How can we check how many merchants use custom checkout?

Reply:
But when we disable or remove experiment DATA-6891.missing_orders_within_GA this methods will not reached anymore. Or we still can get an error in interface AnalyticsTracker?

if (isEcommerceGAEnabled ~~&& this.analytics.hasPayloadLimit(payload)~~) {

Reply:
Ok we will not drop this methods in `analytics.js`.